tests/xls_helpers.py

```diff
@@ -70,9 +82,32 @@ def compare_xls_files(testCase: _ut.TestCase, file_path: str, file_path_expected
                 errors.append(current_error)
                 errors.append(current_error_2)
                 if manual_test:
-                    """ Plot differences in sheet to simplify comparison. """
+                    """Plot differences in sheet to simplify comparison."""
                     plot_dfs_and_differences(df_new, df_expected, sheet)
     if errors:
         if manual_test:
             _plt.show(block=True)
-        raise Exception(errors)
\ No newline at end of file
+        raise Exception(errors)
+
+
```

Comment:
The following two helper functions make it cleaner to run multiple assertions and gather errors before raising all in an ExceptionGroup.